### DISCUSSION TITLE
Datenrate zum WR

### DISCUSSION THREAD

**Discussion by RFDarter** [tbnobody/OpenDTU#1708] [Q&A]
Hi,
Kurze Frage,  wie oft kann das Power Limit geändert werden?
Um eine Sinnvolle Nulleinsleisung zu erreichen sollte es ja mindestens jede Sekunde,  besser natürlich öfter, möglich sein, sonst bekommt man es bei schwankenden Verbrauchern ja nie ausgeregelt.

**Comment by Omega13x** [comment]
Ich habe zwei HM-x00, die werden alle 8Sekunden geregelt. Kürzere Intervalle sind kritisch, da steigen die WR gerne mal aus und schmollen für 10-15Min. Um Schwankungen entgegenzuwirken, habe ich zeitabhängig das Minimum geregelt. Zum Beispiel, Vormittags nie weniger als 250 Watt, so gleicht man die Schwankungen einer Waschmaschine gut aus. Ab 13Uhr dann runter auf 150 Watt Minimum. Für so eine Regelung brauchst du dann aber schon IObroker oder besser noch Home Assistant mit Node-Red.

**Reply by RFDarter** [reply]
Ok, das habe ich mir fast gedacht.
